data/ai/micro_ais/maps/dungeon_mai_comparisons.map

```diff
@@ -0,0 +1,35 @@
+Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu
+Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Cud^Xo, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu
+Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Cud^Xo, Cud^Ebn, Cud^Xo, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu
+Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Cud^Xo, Cud^Xo, Cud^Xo, Cud^Xo, Cud^Xo, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu
+Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Cud^Xo, Cud^Xo, Urb, Ur, Ias, Uu, Uu, Cud^Xo, Cud^Xo, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu
+Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Cud^Xo, Cud^Xo, Cud^Xo, Cud^Xo, Uu, Urb, Uu, Urb, Ias, Ur, Ur, Urb, Urb, Cud^Xo, Cud^Xo, Cud^Xo, Cud^Xo, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu
```

Comment:
The map is decorative but doesn't give a good illustration of the bosses' areas. I assumed that each courtyard would be the bosses' area, but the areas are (radius 5) which the courtyards are (radius 6 with 6 corners removed).

Reply:
The map was made like that to exhibit the case of the elite-class units behavior on units which are in the zone and within or outside their movement range.

Reply:
I'm don't understand. The visual clues on the map are making me think the zones are different to their actual hexes.

What are the castle hexes around the zones meant to mean?

Reply:
The castle hexes around the zones dont serve any role other than to further augment the appearance of an arena. Aesthetics.

Reply:
For the micro ai demo scenarios, I think understanding is more important than aesthetics. You could do with labels on the map, but please communicate the AI's zones to the person viewing the map.